src/helpers/get-user-data.ts

```diff
@@ -97,6 +97,10 @@ export function getUserData (
 
   const user = checkOneUser(users);
 
+  if (typeof user.verifyChanges === "string" && user.verifyChanges[0] === "{") {
+    user.verifyChanges = JSON.parse(user.verifyChanges);
```

Comment:
Maybe this:

```ts
try {
  user.verifyChanges = JSON.parse(user.verifyChanges);
}
```

This ensures that it's only parsed if `user.verifyChanges` is valid json.

Reply:
Agreed. I am not familiar with how to provide better error reporting for this package, just threw an error from catch block (try without catch is not a valid construct in js). If you have better ideas let me know. Thanks